addons/calendar/models/calendar.py

```diff
@@ -1127,7 +1128,7 @@ def get_search_fields(self, order_fields, r_date=None):
         return sort_fields
 
     @api.multi
-    def get_recurrent_ids(self, domain, order=None):
+    def get_recurrent_ids(self, domain, order=None, start=None, stop=None):
```

Comment:
Not sure that this can be merged. As you know, the problem doesn't exist in v14 and workaround for the problem is don't use recurring events with too large repetition number

Reply:
Interesting, I didn't know any of those... 🤔

Reply:
I suppose, it's an irony. 

I just noted why I don't expect much pressure to merge it. Not everything is merged, even clean and useful PR.  If you did it 2 years ago, that would be easier, but not now, unfortunately.

Reply:
> workaround for the problem is using small repetition number for recurring events

But can this actually happen when the calendar is synced with google? That's our problem actually. Won't Google increase the loops again on next sync?

Reply:
@Aurelienvd what do you think?